src/Wrapper/SeparatedVariation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\PlentyMarketsRestExporter\Wrapper;
+
+use FINDOLOGIC\PlentyMarketsRestExporter\RegistryService;
+use FINDOLOGIC\PlentyMarketsRestExporter\Response\Entity\Pim\Property\ImageAttributeValue;
+use FINDOLOGIC\PlentyMarketsRestExporter\Response\Entity\Pim\Variation;
+
+class SeparatedVariation
+{
+    private Variation $variation;
+
+    private RegistryService $registryService;
+
+    public function __construct(
+        Variation $variation,
+        RegistryService $registryService
+    ) {
+        $this->variation = $variation;
+        $this->registryService = $registryService;
+    }
+
+    /**
+     * @param ImageAttributeValue[] $imageAttributeValues
+     * @param array $variationAttributes
```

Comment:
Isn't his a key/value of strings?

```suggestion
     * @param array<string, string> $variationAttributes
```

Reply:
Nop, it hasa value like [0] => '112_150', [1] => '113_140'

Reply:
My point is, if we know what values it may contain, we should definitely typehint it. So it would be `array<int, string>` ot `string[]` then.